tests/python/unittest/test_gluon.py

```diff
@@ -915,6 +915,24 @@ def test_sequential_warning():
         assert len(w) == 1
 
 
+@with_seed()
+def test_dense_backward():
```

Comment:
I guess rename the `test_dense_double_backward` or similar?

Reply:
Can we also move this test to test_higher_order.py so it's easier to track higher order supported operators?

Reply:
Where is this file?